### PR TITLE
Add linguist details in repo page

### PR DESCRIPTION
**Pull request by yg**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

Add linguist bar in repo page

We need to think about colors!

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Screenshot

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by yg** [comment]
@bdougie WDYT on this? for now, I messed up with CSS!

Screenshot

**Comment by filiptronicek** [comment]
Should it be 4+ or +4 languages?

**Comment by yg** [comment]
Ahh, thanks @filiptronicek Good catch! GitHub too doing like that!

Screenshot

**Comment by filiptronicek** [comment]
The color circles are a bit off in Firefox.
screenshot

**Comment by filiptronicek** [comment]
@bdougie @yg What about making the color cicles a bit bigger? I think the're pretty tiny.

**Comment by yg** [comment]
@bdougie The alignment is fine on my screen! I don't know what is the reason! Can you help me with this!

Screenshot

**Comment by filiptronicek** [comment]
@yg have you tried on multiple browsers?

**Comment by bdougie** [comment]
@yg what is your browser and version?

I was able to align things with display flex and align-items center. I can test in my end but want to test is a similar environment as yours.

**Comment by yg** [comment]
I just tested in Browserstack firefox, it is happening!

Screenshot

**Comment by yg** [comment]
@bdougie I'm using `Chrome Version 84.0.4147.45 (Official Build) beta (64-bit)`

File: src/components/Repository.js [hunk 1]
```diff
@@ -83,6 +83,17 @@ function Repository({match}) {
                 </a>
               </em>
             </small>
+            <div style={{marginTop: 20}}>
+              {repository && repository.languages.nodes.map((language, key) => (
+                <span key={key}>
+                  <span style={{color: language.color, fontSize: 25, margin: 0, verticalAlign: "middle"}}>•</span>
```
Comment (bdougie) [inline src/components/Repository.js:89]:
```suggestion
                  <span style={{color: language.color, fontSize: 25, margin: 0, verticalAlign: "sub"}}>•</span>
```

This is this fix to the alignment issue

Comment (bdougie) [reply src/components/Repository.js:89]: I am noticing the octicon alignment has weird results if the incorrect attribute is used.

File: src/components/Repository.js [hunk 2]
```diff
@@ -83,6 +83,17 @@ function Repository({match}) {
                 </a>
               </em>
             </small>
+            <div style={{marginTop: 20}}>
+              {repository && repository.languages.nodes.map((language, key) => (
+                <span key={key}>
+                  <span style={{color: language.color, fontSize: 25, margin: 0, verticalAlign: "sub"}}>•</span>
+                  <span style={{color: "black"}}>{language.name}</span>
```
Comment (bdougie) [inline src/components/Repository.js:90]: move style to a styled-component.

Comment (yg) [reply src/components/Repository.js]: Done! :shipit:

**Comment by yg** [comment]
The mobile view is also good!

**Comment by bdougie** [comment]
@filiptronicek can you gives this one more review to confirm it is good for you.

**Pull request merged**